mcp/src/update-changelog-md.ts

```diff
@@ -0,0 +1,86 @@
+import path from "path";
+import {
+  findAzureSdkRoot,
+  findModuleDirectory,
+} from "./utils/sdk-directory.js";
+import { getJavaSdkChangelogJson } from "./java-sdk-changelog.js";
+import { CallToolResult } from "@modelcontextprotocol/sdk/types";
+import * as fs from "fs";
+
+export async function updateChangelogMd(
+  jarPath: string,
+  groupId: string,
+  artifactId: string,
+): Promise<CallToolResult> {
+  try {
+    const repoRoot = await findAzureSdkRoot(path.dirname(jarPath));
```

Comment:
why do we need to use `path.dirname(jarPath)` instead of `jarPath` (the same as line 17)